packages/hoc-disabled-tooltip/src/disabledTooltip.js

```diff
@@ -19,50 +19,30 @@ function disabledTooltip(wrappedComponentType) {
   const WrappedComponent = props =>
     React.createElement(wrappedComponentType, props);
   return class extends React.Component {
-    constructor(props) {
-      super(props);
-      this.state = {
-        showTooltip: false
-      };
-      this.onMouseOver = this.onMouseOver.bind(this);
-      this.onMouseLeave = this.onMouseLeave.bind(this);
-    }
-
-    onMouseOver(event) {
-      this.setState({ showTooltip: !this.props.noTooltip });
-      return this.props.onMouseOver(event);
-    }
-
-    onMouseLeave(event) {
-      this.setState({ showTooltip: false });
-      return this.props.onMouseLeave(event);
-    }
-
     render() {
+      const WrappedComponentWithTooltip = withTooltip(WrappedComponent);
```

Comment:
It's composing the components on each render cycle... Let's remove this from here.